src/js/views/globals/search/patient-search_views.js

```diff
@@ -90,34 +90,30 @@ const PicklistItem = View.extend({
         {{far "address-card"}}
       </div>
       <div class="patient-search__picklist-item-name-text">
-        <span>{{matchText name search}}{{~ remove_whitespace ~}}</span>
+        <span>{{name}}{{~ remove_whitespace ~}}</span>
       </div>
     </div>
     <div class="patient-search__picklist-item-meta">
       <div class="patient-search__picklist-item-dob u-text--overflow">
         {{formatDateTime birth_date "MM/DD/YYYY"}}
       </div>
-      {{#if hasIdentifiers}}
-        <div class="patient-search__picklist-item-identifier u-text--overflow">
-          {{matchText identifiers.0.value search}}
-        </div>
-      {{/if}}
-      {{#if hasFieldQueryResults}}
-        <div class="patient-search__picklist-item-query u-text--overflow">
-          {{matchText value search}}
-        </div>
-      {{/if}}
       <div class="patient-search__picklist-item-status u-text--overflow">
         {{status}}
       </div>
+      <div class="patient-search__picklist-item-result-value u-text--overflow">
+        {{matchText resultValue search}}
+      </div>
+      <div class="patient-search__picklist-item-result-label u-text--overflow">
+        {{resultLabel}}
+      </div>
     </div>
   `,
   templateContext() {
     return {
       name: `${ this.model.get('first_name') } ${ this.model.get('last_name') }`,
       search: this.state.get('search'),
-      hasIdentifiers: this.collection.hasIdentifiers(),
-      hasFieldQueryResults: this.collection.hasFieldQueryResults(),
+      resultValue: this.model.get('match').value,
+      resultLabel: this.model.get('match').label,
```

Comment:
Do we need to make this safer? Like `this.model.get('match')?.value` or something similar?

Reply:
just use {{ match.value}} in the template

Reply:
Good call, updated to that